server/src/lib/clip.ts

```diff
@@ -252,77 +258,87 @@ export default class Clip {
           console.log("Not converting mpeg/AAC");
        }
 
-      console.log("HEADERS: ", JSON.stringify(headers));
-      const audioOutput = new Transcoder(audioInput)
-        .audioCodec(config.TRANSCODE.AUDIO_CODEC)
-        .format(config.TRANSCODE.FORMAT)
-        .channels(1)
-        //.sampleRate(config.TRANSCODE.SAMPLE_RATE)
-        .on('error', (error: string) => {
-          this.clipSaveError(
-            headers,
-            response,
-            500,
-            `${error} for ${metadata}`,
-            `ffmpeg ${error}`,
-            'clip'
-          );
-          return;
-        })
-        .on('finish', async () => {
-          console.log(`clip written to s3 ${metadata}`);
-
-          await this.model.saveClip({
-            client_id: client_id,
-            localeId: sentence.locale_id,
-            original_sentence_id: sentenceId,
-            path: clipFileName,
-            sentence: sentence.text,
-          });
-          await Awards.checkProgress(client_id, { id: sentence.locale_id });
+      var self = this;
+      fs_promises.mkdtemp(os.tmpdir() + `/${client_id}_${filePrefix}`)
+          .then(function(tmp_dir_name: string) {
+             const path = tmp_dir_name + `/output.${config.TRANSCODE.FORMAT}`;
+             console.log(path);
```

Comment:
we probably don't need these console logs in production, do you think?

Reply:
Yes indeed, we don't need the logs.  The PR is not final as I wanted to have eyes on my require/import problem.

Reply:
You can click on `Convert to draft` under the reviewers in the top right to flag it's still a draft PR. Of course, we might interpret that as "don't review yet", so add a message saying you're asking for early review when you do that.